money-maven-web/src/components/LineChart.jsx

```diff
@@ -0,0 +1,173 @@
+    }
+  };
+
+  const getLabels = (startDate, endDate) => {
+    const start = startDate ? moment(startDate) : moment().startOf("year");
+    const end = endDate
+      ? moment(endDate).endOf("month")
+      : moment().endOf("month");
+    const numMonths = end.diff(start, "months") + 1;
+    return Array(numMonths)
+      .fill()
+      .map((_, i) => start.clone().add(i, "months").format("MMM"));
+  };
+
+  const dataLine = {
+    labels: getLabels(startDate, endDate),
+    datasets: [
+      {
+        type: "line",
+        label: "Expenses",
+        backgroundColor: "red",
+        tension: 0.3,
+        data: expenses.map(
+          (selectedExpenses) => selectedExpenses.expenseAmount
+        ),
+      },
+    ],
+  };
+  return (
+    <Container>
+      <Form.Label>Line Chart</Form.Label>
+      <Row className="row justify-center">
+        <Row className="table-cell">
+          <Row className="table-filter">
+            <Col className="table-filter--size">
+              <Form.Group>
+                <Col>
+                  <Form.Label>Date from:</Form.Label>
+                  <DateTimePicker
+                    value={startDate}
+                    name="startDate"
+                    format="yyyy-MM-dd"
+                    className="table-filter--date"
+                    onChange={handleStartDateChange}
+                    disableClock={true}
+                    calendarIcon={<FaCalendarAlt />}
+                  />
+                </Col>
+              </Form.Group>
+            </Col>
+          </Row>
+          <Row className="table-filter">
+            <Col className="table-filter--size">
+              <Form.Group>
+                <Col>
+                  <Form.Label>Date until:</Form.Label>
+                  <DateTimePicker
+                    value={endDate}
+                    name="endDate"
+                    format="yyyy-MM-dd"
+                    onChange={handleEndDateChange}
+                    disableClock={true}
+                    calendarIcon={<FaCalendarAlt />}
+                    isInvalid={
+                      startDate &&
+                      endDate &&
+                      moment(startDate).isAfter(moment(endDate))
+                        ? "is-invalid"
+                        : "Start date cannot be after end date"
+                    }
+                    className="table-filter--date"
+                  />
+                </Col>
+              </Form.Group>
+            </Col>
+          </Row>
+        </Row>
+        {!expenses && <tr>Data not exist, wrong date format</tr>}
+        <Line data={dataLine} />
+        <Table class="table table">
+          <tbody>
+            <td class="table-active">
+              <Row className="table-filter--date">
+                <td class="table-Secondary">
+                  Selected period expenses : € {selectedExpenses}
+                </td>
+              </Row>
+            </td>
+          </tbody>
+        </Table>
```

Comment:
ar cia tikrai viskas gerai nes is pirmo zvilgsnio atrodo jog jei nera expenses yra tr elementas be jokio table ji apglebencio ir tada 2 td elementai vienas kitame be jokio tr elemento